benchmarks/src/main/resources/org/elasticsearch/benchmark/index/mapper/sample_documents.json

```diff
@@ -0,0 +1,25 @@
+{"@timestamp": 893964617, "client.ip": "40.135.0.0", "http.request.method": "GET", "url.path": "/images/hm_bg.jpg HTTP/1.0", "http.response.status_code": 200, "http.response.bytes": 24736}
```

Comment:
Should we generate these in some form as well like we do in the other benchmark maybe? This seems a little noisy when we already have code to generate documents anyway?